File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// AirQualityIndex SDK exists test

require_once __DIR__ . '/../airqualityindex_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = AirQualityIndexSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# AirQualityIndex SDK exists test

require "minitest/autorun"
require_relative "../AirQualityIndex_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = AirQualityIndexSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
